src/sentry/static/sentry/app/views/settings/organizationProjects/index.tsx

```diff
@@ -12,43 +15,46 @@ import Pagination from 'app/components/pagination';
 import {Panel, PanelBody, PanelHeader, PanelItem} from 'app/components/panels';
 import Placeholder from 'app/components/placeholder';
 import ProjectListItem from 'app/views/settings/components/settingsProjectItem';
-import SentryTypes from 'app/sentryTypes';
 import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
 import routeTitleGen from 'app/utils/routeTitle';
 import space from 'app/styles/space';
 import withOrganization from 'app/utils/withOrganization';
 import {IconAdd} from 'app/icons';
+import {decodeScalar} from 'app/utils/queryString';
 
 import ProjectStatsGraph from './projectStatsGraph';
 
 const ITEMS_PER_PAGE = 50;
 
-class OrganizationProjects extends AsyncView {
-  static propTypes = {
-    organization: SentryTypes.Organization,
-  };
+type Props = {
+  organization: Organization;
+  location: Location;
+} & RouteComponentProps<{orgId: string}, {}>;
+
+type ProjectStats = Record<string, Required<Project['stats']>>;
 
+type State = AsyncView['state'] & {
+  projectList: Project[] | null;
+  projectListPageLinks: string | null;
+  projectStats: ProjectStats | null;
+};
+
+class OrganizationProjects extends AsyncView<Props, State> {
   static contextTypes = {
-    router: PropTypes.object.isRequired,
+    router: PropTypes.object,
```

Comment:
do we still need this contextTypes? it doesn't seem to be being used

Reply:
I'll remove it in that follow up pull request.